upnp.py

```diff
@@ -1,3 +1,4 @@
+"""Adapted the code to make a Huawei version that uses dslforum schemas"""
 #!/bin/python
```

Comment:
The [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) should always come first